### PR TITLE
Add query node for multi-validator build

### PR DESCRIPTION
**Pull request by taobun**

### CODE REVIEW

File: chain/cmd/request/main.go [hunk 1]
```diff
@@ -81,14 +81,14 @@ func main() {
 					// BTC parameter
 					params, _ := hex.DecodeString("00000000")
 					// Send request by code hash
-					fmt.Println(tx.SendTransaction(zoracle.NewMsgRequest(codeHash, params, 4, tx.Sender()), flags.BroadcastBlock))
+					fmt.Println(tx.SendTransaction(zoracle.NewMsgRequest(codeHash, params, 6, tx.Sender()), flags.BroadcastBlock))
```
Comment (sorawit) [inline chain/cmd/request/main.go:84]: Why is this changed?

**Review by sorawit — CHANGES_REQUESTED** [review]

File: chain/cmd/request/main.go (hunk 1 above)
Comment (taobun) [reply chain/cmd/request/main.go:84]: I tried to run in local mode, 4 block time duration is not enough to get data and report in time. After investigation, I found report miss in a little of time after change it to 6 blocks. All providers report successfully.

Comment (sorawit) [reply chain/cmd/request/main.go:84]: Make sense.

**Review by sorawit — APPROVED** [review]

**Review by prin-r — APPROVED** [review]
